tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table_test.py

```diff
@@ -0,0 +1,201 @@
+            INSERT INTO
+              `{{project_id}}.{{dataset_id}}.concept` (
+                concept_id,
+                concept_code,
+                concept_name)
+            VALUES
+              (1585266, '', 'PII State: CA'),
+              (1585847, 'SexAtBirth_Female', '')
+            """).render(project_id=self.project_id, dataset_id=self.dataset_id)
+
+        # test data for observation_ext table
+        observation_ext_data_query = self.jinja_env.from_string("""
+            DROP TABLE IF EXISTS
+              `{{project_id}}.{{dataset_id}}.observation_ext`;
+            CREATE TABLE
+              `{{project_id}}.{{dataset_id}}.observation_ext` (
+                observation_id INT64,
+                src_id STRING);
+            INSERT INTO
+              `{{project_id}}.{{dataset_id}}.observation_ext` (
+                observation_id,
+                src_id
+              )
+            VALUES
+              (111, 'PPI/PM'),
+              (222, 'PPI/PM'),
+              (333, 'PPI/PM'),
+              (444, 'PPI/PM')
+                    """).render(project_id=self.project_id,
+                                dataset_id=self.dataset_id)
+
+        # run query to create person_ext
+        person_ext_query = PERSON_EXT_TABLE_QUERY.render(
+            project=self.project_id, dataset=self.dataset_id)
+
+        self.load_test_data([
+            person_data_query, observation_data_query, concept_data_query,
+            observation_ext_data_query, person_ext_query
+        ])
+
+    def test_identifying_field_suppression(self):
+        """
+        Tests that the specifications for SANDBOX_RECORDS_QUERY and DROP_RECORDS_QUERY perform as designed.
+
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.
+        """
+
+        tables_and_counts = [{
+            'fq_table_name':
+                '.'.join([self.fq_dataset_name, 'person_ext']),
+            'fields': [
+                'person_id', 'src_id', 'state_of_residence_concept_id',
+                'state_of_residence_source_value', 'sex_at_birth_concept_id',
+                'sex_at_birth_source_concept_id', 'sex_at_birth_source_value'
+            ],
+            'loaded_ids': [123, 345, 678, 910],
+            'cleaned_values': [(123, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (345, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (678, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female'),
+                               (910, 'PPI/PM', 1585266, 'PII State: CA',
+                                45878463, 1585847, 'SexAtBirth_Female')]
+        }]
+
+        self.default_test(tables_and_counts)
```

Comment:
@amcgrenera-vumc You should add a `tearDown(self):` function to this test.  It isn't cleaning up after itself because it is not relying on defining all the fully qualified tables required to set this test up.  Meaning it leaves a dirty test environment for the next test.  And the next test shouldn't have to worry about cleaning up from the previous test.  Your SQL statements `DROP TABLE IF EXISTS...` are symptomatic of the missing `tearDown()`.  Consider adding:

```suggestion
        self.default_test(tables_and_counts)
            
    def tearDown(self):
        tables = ['observation', 'observation_ext', 'concept', 'person']
        for table in tables:
            self.client.delete_table('.'.join([self.fq_dataset_name, table]), not_found_ok=True)
          
        super().tearDown() 
```

Reply:
good catch, I will add this in, thank you!